telegramBot.js

```diff
@@ -0,0 +1,200 @@
+const fs = require('fs')
+const axios = require('axios')
+const request = require('request')
+const { Telegraf } = require('telegraf')
+
+class TelegramBot {
+    constructor( teleToken ){
+
+        //URLs for http requests
+        this.baseURL = `https://api.telegram.org/bot${teleToken}/`
+        this.updateURL = `${this.baseURL}getUpdates`
+
+        this.bot = new Telegraf(teleToken);
+
+        this.initialzied = false;
+        
+        // allows bot to obtain chat id to send messages and files
+        this.bot.command('init', (ctx) =>{
```

Comment:
do we really need the init command ? because you just cache the `chatId` and `chatTitle`,  this information could be retrieved from any command context object

Reply:
The /init command is a failsafe. It doesn't hurt to have it